server/migrations/20181107082847-create-rating.js

```diff
@@ -0,0 +1,41 @@
+export default {
```

Comment:
Why do we have Ratings migrations here?

Reply:
I lost the file somehow, so I put it back in the project because I needed to use it for a migration.